Weatherman.py

```diff
@@ -0,0 +1,245 @@
+import os
+import sys
+
+
+def weatherMonth(year, month, res):
```

Comment:
the function name should be lowercase and use `_` 
`weatherMonth` irrelevant name function name should be `verb`